unified-scheduler-pool/src/lib.rs

```diff
@@ -1135,6 +1222,59 @@ impl UsageQueueLoader {
     }
 }
 
+#[derive(Debug)]
+enum UsageQueueLoader {
+    OwnedBySelf {
+        usage_queue_loader_inner: UsageQueueLoaderInner,
+    },
+    SharedWithBankingStage {
```

Comment:
I'm trying to recall the reason this is "shared with banking stage". The usage queue is on the BankingStageHelper, which is held by both scheduler & workers - why do we need it on the workers again?

Reply:
Hope this doc rework should help future you recall the reason..?: 4e7f8e3b8aea91e8bc0632acb07d7e090e364499